CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Version number changes (major.minor.micro) in this package denote the following:
 - A minor version will increase if one or more packages contained in the Docker image add new, backwards-compatible features, or if a new package is added to the Docker image.
 - A major version will increase if there are any backwards-incompatible changes in any of the packages contained in this Docker image, or any other backwards-incompabile changes in the execution environment.
 
+## Unreleased
+### Removed
+- Remove conda install of gcc. This prevented use of the environment file in OS X, and isn't necessary for any other packages.
```

Comment:
This should be more specific. `conda` has separate packages for `gcc` and `libgcc`. Usually, `gcc` is used to build stuff and then you just specify `libgcc` as a run-time dependence. See my [recipe](https://github.com/conda-forge/staged-recipes/pull/2628) for XGBoost, for example.